interfaces/builtin/basedeclaration.go

```diff
@@ -154,6 +154,8 @@ plugs:
   snapd-control:
     allow-installation: false
     deny-auto-connection: true
+  unity8:
+    allow-installation: false
```

Comment:
Please add a test to basedeclaration_test.go for this. Look at snapd-control tests there for some inspiration.